webapp-django/crashstats/crashstats/templates/crashstats/home.html

```diff
@@ -61,7 +61,7 @@ <h2 id="homepage-heading">{{ product }} {% if version %}{{ version }}{% endif %}
                             </a>
                         </li>
                         <li>
-                            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;versions={{ version }}&amp;crash_type=plugin&amp;days={{days}}">
+                            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;versions={{ version }}&amp;process_type=plugin&amp;days={{days}}">
```

Comment:
Perhaps it's time to fix this. Perhaps not as part of this PR. You can decide that. 

We currently have a helper function called `urlencode` which sucks because that's not really what it does. What it does is "url quote" or "url escape". 

Ideally I think the template code should look like this:
```
 <a href="{{ url('topcrashers:topcrashers') }}?{{ urlencode(product=product, versions=version, process_type='plugin', days=days) }}">
```
don't you think?